satpy/etc/readers/seviri_l2_grib.yaml

```diff
@@ -152,3 +187,123 @@ datasets:
     parameter_number: 9
     units: ""
     long_name: active_fire_classification
+
+  aerosol_optical_thickness_vis06:
+    name: aerosol_optical_thickness_vis06
+    resolution: 3000
+    file_type: grib_seviri_aes
+    parameter_number: 20
+    units: "um"
+    long_name: aerosol_optical_thickness_vis06
+
+  aerosol_optical_thickness_vis08:
+    name: aerosol_optical_thickness_vis08
+    resolution: 3000
+    file_type: grib_seviri_aes
+    parameter_number: 21
+    units: "um"
+    long_name: aerosol_optical_thickness_vis08
+
+  aerosol_optical_thickness_vis16:
+    name: aerosol_optical_thickness_vis16
+    resolution: 3000
+    file_type: grib_seviri_aes
+    parameter_number: 22
+    units: "um"
+    long_name: aerosol_optical_thickness_vis06
+
+  angstroem_coefficient:
+    name: angstroem_coefficient
+    resolution: 3000
+    file_type: grib_seviri_aes
+    parameter_number: 23
+    units: ""
+    long_name: angstroem_coefficient
+
+  aes_quality:
+    name: aes_quality
+    resolution: 3000
+    file_type: grib_seviri_aes
+    parameter_number: 192
+    units: ""
+    long_name: aes_quality
+
+  cloud_top_height:
+    name: cloud_top_height
+    resolution: 3000
+    file_type: grib_seviri_cth
+    parameter_number: 2
+    units: Pa
+    long_name: cloud_top_height
+
+  cloud_top_quality:
+    name: cloud_top_quality
+    resolution: 3000
+    file_type: grib_seviri_cth
+    parameter_number: 3
+    units: ""
+    long_name: cloud_top_quality
+
+  vis_refl_06:
+    name: vis_refl_06
+    resolution: 3000
+    file_type: grib_seviri_crm
+    parameter_number: 9
+    units: "%"
+    long_name: vis_refl_06
+
+  vis_refl_08:
+    name: vis_refl_08
+    resolution: 3000
+    file_type: grib_seviri_crm
+    parameter_number: 10
+    units: "%"
+    long_name: vis_refl_08
+
+  vis_refl_16:
+    name: vis_refl_16
+    resolution: 3000
+    file_type: grib_seviri_crm
+    parameter_number: 11
+    units: "%"
+    long_name: vis_refl_16
+
+  nir_refl_39:
+    name: nir_refl_39
+    resolution: 3000
+    file_type: grib_seviri_crm
+    parameter_number: 12
+    units: "%"
+    long_name: nir_refl_39
+
+  num_accumulations:
+    name: num_accumulations
+    resolution: 3000
+    file_type: grib_seviri_crm
+    parameter_number: 6
+    units: ""
```

Comment:
I think it is semi-standard in Satpy to use `"1"` for unitless fields.

Reply:
ok will update

Reply:
ok, done